athena-elasticsearch/src/main/java/com/amazonaws/connectors/athena/elasticsearch/ElasticsearchMetadataHandler.java

```diff
@@ -83,6 +87,17 @@
     // A Map of the domain-names and their respective endpoints.
     private Map<String, String> domainMap;
 
+    /**
+     * Key used to store shard information in the Split's properties map (later used by the Record Handler).
+     */
+    private static final String SHARD_KEY = "shard";
+    /**
+     * Value used in combination with the shard ID to store shard information in the Split's properties map (later
+     * used by the Record Handler). The completed value is sent as a request preference to retrieve a specific shard
+     * from the Elasticsearch instance (e.g. "_shards:5" - retrieve shard number 5).
+     */
+    private static final String SHARD_VALUE = "_shards:";
```

Comment:
probably needs to be protected so that the record handler can reference it?

Reply:
Not needed in Record Handler. value is extracted using ElasticsearchMetadataHandler.SHARD_KEY as a key on the split's properties map and passed as is to the request's preferences argument.